### PR TITLE
update release dates on milestones. starting release scripts.

### PR DESCRIPTION
**Pull request by shawncatz**

### CODE REVIEW

File: release/prepare.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+for r in ${repos}; do
+    if [ ! -e ${r} ]; then
+        git clone git@github.com:revel/${r}
+    fi
+
+    echo ">>> ${r} <<<"
+    (
+        cd ${r}
+        git checkout develop
+        git merge master
```
Comment (notzippy) [inline release/prepare.sh]: On merge failure what happens ?

File: release/changelog.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+echo "# CHANGELOG"
+echo
+echo "## v${ver}"
+echo
+
+for r in ${repos}; do
+    echo "[[$r](https://github.com/revel/$r)]"
+    echo
+    lines=$(git --no-pager -C ${src}/${r} log --oneline master..develop)
```
Comment (notzippy) [inline release/changelog.sh]: Rather then using the commit log, we should be looking to fetch the PR data, and issues from github https://github.com/skywinder/github-changelog-generator

**Review by notzippy — CHANGES_REQUESTED** [review]

File: release/changelog.sh (hunk 2 above)
Comment (shawncatz) [reply release/changelog.sh]: yeah, this was just a first pass, I planned on integration with the generator, just needed to get things working first.

File: release/prepare.sh (hunk 1 above)
Comment (shawncatz) [reply release/prepare.sh]: BOOM!

File: release/changelog.sh (hunk 2 above)
Comment (shawncatz) [reply release/changelog.sh]: I will work on that once I've got the other stuff nailed down... testing the release process right now

**Comment by shawncatz** [comment]
@brendensoares @notzippy are we going to try to do the release tomorrow night? I think this is ready to try.
Or we can try Thursday... whichever works for you both

**Comment by brendensoares** [comment]
Well we did a release with these scripts and it went well overall. I'd say merge it. We can expand on the work you did for Go-based tools as well.

Thanks @shawncatz

**Review by brendensoares — APPROVED** [review]

**Pull request merged**